lib/danger/danger_core/plugins/dangerfile_git_plugin.rb

```diff
@@ -114,5 +114,13 @@ def commits
     def diff_for_file(file)
       modified_files.include?(file) ? @git.diff[file] : nil
     end
+
+    # @!group Git Metadata
+    # Statistics for a specific file in this diff
+    # @return {:insertions => [Fixnum], :deletions => [Fixnum]} or nil
```

Comment:
I'm not a rubyist and am only guessing at the appropriate format for the `@return` annotation.